src/py42/clients/alerts.py

```diff
@@ -136,3 +136,32 @@ def get_aggregate_data(self, alert_id):
             :class:`py42.response.Py42Response`
         """
         return self._alert_service.get_aggregate_data(alert_id)
+
+    def get_all_alert_details(self, query, ascending=True):
+        """
+        Helper method that combines `.get_all_pages()` and `.get_details()`.
```

Comment:
I think this should read "Helper method that combines `.search_all_pages()` and `.get_details()`."

Reply:
updated to add :func: for doc formatting

Reply:
and correct function name..